crates/iota-data-ingestion/src/workers/mod.rs

```diff
@@ -5,6 +5,8 @@
 mod archival;
 mod blob;
 mod kv_store;
-pub use archival::{ArchivalConfig, ArchivalReducer, ArchivalWorker};
+mod relay;
+pub use archival::{ArchivalConfig, ArchivalReducer};
```

Comment:
nit: maybe add an empty line between the mod definitions and pub use statements that it is better separated

Reply:
added with https://github.com/iotaledger/iota/pull/5925/commits/a7b8470894104e15ce8f6ac240ad5efe42369d18